packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -623,14 +632,15 @@ contract OPContractsManager is ISemver {
             if (lastGameConfig >= gameTypeInt) revert InvalidGameConfigs();
             lastGameConfig = gameTypeInt;
 
-            // Grab the FDG from the SystemConfig.
-            IFaultDisputeGame fdg = IFaultDisputeGame(
+            // Grab the permissioned and fault dispute games from the SystemConfig.
+            // We keep the FDG type as it reduces casting below.
+            IFaultDisputeGame pdg = IFaultDisputeGame(
                 address(
                     getGameImplementation(getDisputeGameFactory(gameConfig.systemConfig), GameTypes.PERMISSIONED_CANNON)
```

Comment:
The old `fdg` varname was misleading since this the `PERMISSIONED_CANNON` game type is being retrieved.